src/components/Dashboard/ManageUsersPage/AddUserPage.js

```diff
@@ -159,6 +172,41 @@ const addUserMutation = gql`
   }
 `;
 
-export default graphql(addUserMutation, {
-  name: 'addUserMutation',
-})(AddUserPage);
+const addUserWithNewCommunityMutation = gql`
```

Comment:
Nice, this makes a lot of sense because why would someone try to make a community without a user?